Creating a chat with numeric only crashes the app

### Description

```
panic: runtime error: slice bounds out of range [2:1]

goroutine 899 [running, locked to thread]:
github.com/status-im/status-go/protocol/identity/alias.GenerateFromPublicKeyString({0xc0008fbfb4, 0xc000979e20})
	/Users/anastasiya/status-desktop/vendor/status-go/protocol/identity/alias/generate.go:34 +0xdc
github.com/status-im/status-go/protocol.GenerateAlias(...)
	/Users/anastasiya/status-desktop/vendor/status-go/protocol/messenger.go:4064
github.com/status-im/status-go/mobile.GenerateAlias(...)
	/Users/anastasiya/status-desktop/vendor/status-go/mobile/status.go:638
main.GenerateAlias(0x0)
	/Users/anastasiya/status-desktop/vendor/status-go/build/bin/statusgo-lib/main.go:72 +0x25
make: *** [run-macos] Error 2
```

1. open status
2. join public chat with name with numbers only (for ex # 2)

--- a/src/app_service/service/contacts/async_tasks.nim
+++ b/src/app_service/service/contacts/async_tasks.nim
@@ -8,8 +8,6 @@ include ../../../app/core/tasks/common
 # Async lookup ENS contact
 #################################################
 
-const PK_LENGTH_0X_INCLUDED = 132
-
 type
   LookupContactTaskArg = ref object of QObjectTaskArg
     value: string

--- a/src/app_service/service/contacts/service.nim
+++ b/src/app_service/service/contacts/service.nim
@@ -16,6 +16,8 @@ import ../../../backend/utils as status_utils
 
 export contacts_dto, status_update_dto, contact_details
 
+const PK_LENGTH_0X_INCLUDED = 132
+
 include async_tasks
 
 logScope:
@@ -211,6 +213,16 @@ QtObject:
 
     result = self.fetchContact(id)
     if result.id.len == 0:
+      if(not id.startsWith("0x")):
+        debug "id is not in a hex format"
+        return
+      
+      var num64: int64
+      let parsedChars = parseHex(id, num64)
+      if(parsedChars != PK_LENGTH_0X_INCLUDED):
+        debug "id doesn't have expected lenght"
+        return 
+
       let alias = self.generateAlias(id)
       let identicon = self.generateIdenticonURL(id)
       result = ContactsDto(

